### ISSUE TITLE
UX bug when editing alt text

### ISSUE THREAD

**Issue by Gadgetoid** [tooot-app/app#667]
After a few weeks of using the app to run a microblog with images and alt text I would then embed into my main blog, I've just noticed that image the alt text for a great deal of my toots has been lost.

A little sleuthing pins this on a trap in the alt text editing UX. There's a "Down Arrow" on the left which closes this window, and a little "Disk" icon on the right which *saves* the edits.

If you edit text, click the down arrow, then open up the edit window again the text will still be visible- giving the illusion that it has been saved. Posting the toot, however, will not include this alt text unless it's saved *explicitly* with the little disk icon.

To reproduce:

1. Create a new toot
2. Add an image
3. Add some alt text
4. Hit the down arrow on the left
5. Open the alt text edit window again, and observe the text is still there
6. Hit the down arrow again
7. Post the toot
8. View the toot and notice a suspicious lack of alt text

To fix this:

* Opening the alt text editor, making some edits and clicking the down arrow should probably throw up a prompt along the lines of "Are you sure you want to abandon your changes?" since this is effectively what it does.

* Or - the arrow/disk could just be a single action, requiring the text to be saved if the editor is closed.

* Or - all alt text should be saved automatically at the point that the toot is posted, regardless of whether it's done explicitly for each individual image.
